hack/package.txt

```diff
@@ -0,0 +1,15 @@
+gardener/gardener/extensions/
```

Comment:
I would rather go for an exclusion list. If folks add "exported/public" packages, they won't think about adding it here. 
Let's rather list all packages that shall be excluded, because we know, they are not supposed to be used elsewhere. Then we can cut down on false-positives later on to reduce noice.

Reply:
Writing and exclusion will make the list very exhaustive Since there is less number pkg that is exported compared to those which are not.